doc/build/dialects/index.rst

```diff
@@ -43,47 +43,49 @@ External Dialects
-| Apache Solr                           | sqlalchemy-solr_      |
-+---------------------------------------+-----------------------+
-| CrateDB                               | crate-python_         |
-+---------------------------------------+-----------------------+
-| EXASolution                           | sqlalchemy_exasol_    |
-+---------------------------------------+-----------------------+
-| Elasticsearch (readonly)              | elasticsearch-dbapi_  |
-+---------------------------------------+-----------------------+
-| Firebird                              | sqlalchemy-firebird_  |
-+---------------------------------------+-----------------------+
-| Google BigQuery                       | pybigquery_           |
-+---------------------------------------+-----------------------+
-| Google Sheets                         | gsheets_              |
-+---------------------------------------+-----------------------+
-| IBM DB2 and Informix                  | ibm-db-sa_            |
-+---------------------------------------+-----------------------+
-| Microsoft Access (via pyodbc)         | sqlalchemy-access_    |
-+---------------------------------------+-----------------------+
-| Microsoft SQL Server (via python-tds) | sqlalchemy-tds_       |
-+---------------------------------------+-----------------------+
-| MonetDB                               | sqlalchemy-monetdb_   |
-+---------------------------------------+-----------------------+
-| SAP Hana                              | sqlalchemy-hana_      |
-+---------------------------------------+-----------------------+
-| SAP Sybase SQL Anywhere               | sqlalchemy-sqlany_    |
-+---------------------------------------+-----------------------+
-| Snowflake                             | snowflake-sqlalchemy_ |
-+---------------------------------------+-----------------------+
-| Teradata Vantage                      | teradatasqlalchemy_   |
-+---------------------------------------+-----------------------+
++---------------------------------------+-------------------------+
+| Database                              | Dialect                 |
++=======================================+=========================+
+| Amazon Redshift (via psycopg2)        | sqlalchemy-redshift_    |
++---------------------------------------+-------------------------+
+| Apache Drill                          | sqlalchemy-drill_       |
++---------------------------------------+-------------------------+
+| Apache Druid                          | pydruid_                |
++---------------------------------------+-------------------------+
+| Apache Hive and Presto                | PyHive_                 |
++---------------------------------------+-------------------------+
+| Apache Solr                           | sqlalchemy-solr_        |
++---------------------------------------+-------------------------+
+| CockroachDB                           | sqlalchemy-cockroachdb_ |
```

Comment:
I just noticed that the whole table needed to be reformatted.
Could you use a shorter placeholder here? like `sa-cockroachdb_` so we avoid 40+ changes?

Reply:
Thanks for the review. If I understand correctly, that's not just placeholder text -- isn't it the text that shows up in the final document? So the link name will be different that the actual name of the dialect package.

I can make this change, but I'm just checking to make sure I understand.

Reply:
You are right, sorry. I thought that the actual text was provided below, but it seem that the text in the table is indeed what is rendered in the final document.

Since it's more appropriate using the full package name, `sqlalchemy-cockroachdb_` and it requires changing the table anyway, I think we could be a bit generous and have a few more spaces in the right columns

Reply:
That sounds good. Done.

Reply:
thansk, sorry for the back and forth